polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -783,6 +1078,23 @@ $loading-panel-height: 53px;
 }
 
 .Table-unselectable {
+  // for a consistency we calculate indentation for nested rows the same way as for selectable table
+  // stylelint-disable -- Polaris component custom properties
+  --pc-index-table-checkbox-width: var(--p-space-500);
+  // checkbox width is bigger for 2px for small devices
+  --pc-index-table-checkbox-width-sm: calc(
+    var(--pc-index-table-checkbox-width) + var(--p-width-050)
+  );
+  --pc-index-table-checkbox-padding-left: var(--p-space-300);
+  --pc-index-table-checkbox-child-offset: calc(
+    var(--pc-index-table-checkbox-width) +
+      var(--pc-index-table-checkbox-padding-left)
+  );
+  --pc-index-table-checkbox-child-offset-sm: calc(
+    var(--pc-index-table-checkbox-width-sm) +
+      var(--pc-index-table-checkbox-padding-left)
+  );
+  // stylelint-enable -- Polaris component custom properties
```

Comment:
If this is the same as in the case of [selectable rows](https://github.com/Shopify/polaris/pull/10599/files?show-viewed-files=true&file-filters%5B%5D=#diff-e9d0c2a006c030f315115b92e46c3b44667bef0b5e909e1b3af852e6eb2b605bR269-R285) - can the custom properties and the final offset value calculation be defined in one place for all since it's being reused? For example, defined at the [.Table](https://github.com/Shopify/polaris/pull/10599/files?show-viewed-files=true&file-filters%5B%5D=#diff-e9d0c2a006c030f315115b92e46c3b44667bef0b5e909e1b3af852e6eb2b605bL93) level and then used in `&.TableRow-child` and `.Table-unselectable`? If we can take [Frame](https://github.com/Shopify/polaris/blob/64ee750393a67ec271908e597403cc2035e35660/polaris-react/src/components/Frame/Frame.scss) as an example -  the custom property is defined at `.Frame` and then [used](https://github.com/Shopify/polaris/blob/64ee750393a67ec271908e597403cc2035e35660/polaris-react/src/components/Frame/Frame.scss#L95-L97) further on in `.NavigationDismiss`.